src/governance/object.cpp

```diff
@@ -123,6 +123,13 @@ bool CGovernanceObject::ProcessVote(const CGovernanceVote& vote, CGovernanceExce
         return false;
     }
 
+    if (dmn->pdmnState->IsBanned()) {
```

Comment:
what if someone will DOS node and get it temporary banned? Does it mean that its vote would not be counted too?

Seems as legit vector for attack.

Reply:
There is already an incentive to DoS MNs to kick them off the payment queue to get more rewards. Or to disrupt quorums. I doubt ignoring invalid votes would affect incentives in any noticeable way but if you feel like it is better to avoid that here is an alternative solution https://github.com/dashpay/dash/pull/5584

Reply:
I don't think this is a good change.  Does this mean a MN that enters PoSe ban status will need to recast its votes upon revival?  Also, note that this change will make it harder for proposals to pass and create a degree of randomness in the votes especially at the end of the cycle where a marginal proposal may fail to pass due to a mass outage.